app/scripts/lib/auth-errors.js

```diff
@@ -492,6 +492,10 @@ define(function (require, exports, module) {
     RECOVERY_KEY_REQUIRED: {
       errno: 1059,
       message: t('Recovery key required')
+    },
+    TOKEN_VERIFICATION_CODE_REQUIRED: {
+      errno: 1060,
+      message: t('Valid code required')
```

Comment:
I know we discussed this in IRC already, but noting it officially in the PR too. As these new strings missed the cut, we need to do something to limit their usage by either locale or just to Mozilla email addresses or whatever.

Reply:
Discussed with @vbudhram in IRC, this will be handled by #6429.